docs/vscode.md

```diff
@@ -0,0 +1,105 @@
+# Visual Studio Code Installation
+
+Install nodeJs, yarn and java according to your system:
+- Windows: [./windows-setup.md](./windows-setup.md)
+- Linux / Windows Subsystem for Linux: [./wsl-setup.md](./wsl-setup.md)
+
+Download and Install Visual Studio Code or use chocolatey:
+
+https://code.visualstudio.com/Download  
+https://code.visualstudio.com/docs/setup/linux  
+or
+```
+choco install vscode
```

Comment:
Imo not neccessary. People who are interested in installing via chocolatey, scoop etc. will know how to do it. If you still prefer to leave it then also add scoop as I personally am a scoop person 🙈 
```cmd
scoop bucket add extras
scoop install vscode
```

Reply:
We recommended chocolate in the windows-setup.md file. 
I don't know who wrote this, but I think it is better to stay with only one package manager. 

.. and I admit I put the commands because I am using it ^^